io_antarctica_scene/blender_manifest.toml

```diff
@@ -0,0 +1,21 @@
+schema_version = "1.0.0"
+
+id = "io_antarctica_scene"
+version = "4.0.0"
+name = "SuperTuxKart Exporter Tools"
+tagline = "Export items to SuperTuxKart (karts/tracks/materials)"
+maintainer = "Jean-Manuel Clemencon, Joerg Henrichs, Marianne Gagnon, Richard Qian"
+
+type = "add-on"
+website = "https://supertuxkart.net/Main_Page.html"
```

Comment:
It would be nice if instead of the main page we could point to a page on the website that's more directly about the Blender addon. We may not have such a webpage right now and that's not a blocking issue, but it would be nice to have.